Public broadcasting of Latvia.js

```diff
@@ -0,0 +1,335 @@
+{
+	"translatorID": "dab3903a-0289-46f8-a74a-eccf7bdb86b6",
+	"label": "Public broadcasting of Latvia",
+	"creator": "Mārtiņš Bruņenieks",
+	"target": "^https?://(www\\.)?lsm\\.lv/raksts/",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2019-10-22 10:41:39"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2019 Mārtiņš Bruņenieks
+	
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+	if (url.includes('/raksts/')) {
+		return "newspaperArticle";
+	}
+	return false;
+}
+
+
+function doWeb(doc, url) {
+	if (detectWeb(doc, url) == "newspaperArticle") {
+		scrape(doc, url);
+	}
+}
+
+function formatDate(date) {
+	return date.toISOString().split('T')[0];
+}
+
+function scrape(doc, url) {
+	var translator = Zotero.loadTranslator('web');
+	// Embedded Metadata
+	translator.setTranslator('951c027d-74ac-47d4-a107-9c3069ab7b48');
+
+	translator.setHandler('itemDone', function (obj, item) {
+		item.publicationTitle = ZU.xpathText(doc, '//section[@class="globalnav__item globalnav__item-sitename"]');
```

Comment:
Can this be anything else then `Latvijas Sabiedriskie Mediji`?

All test cases have the same value and maybe we should just set it automatic to this.